src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/ContextMenuStripGroupCollectionTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+namespace System.Windows.Forms.Design.Tests;
+
+public class ContextMenuStripGroupCollectionTests
+{
+    [Fact]
+    public void Indexer_ShouldCreateNewGroupIfKeyDoesNotExist()
+    {
+        ContextMenuStripGroupCollection collection = new ContextMenuStripGroupCollection();
```

Comment:
Please use target type new. Same suggestion for the other tests as well
```suggestion
        ContextMenuStripGroupCollection collection = new();
```